var/spack/repos/builtin/packages/montage/package.py

```diff
@@ -0,0 +1,62 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Montage(MakefilePackage):
+    """Montage is a toolkit for assembling Flexible Image Transport System
+        (FITS) images into custom mosaics."""
+
+    homepage = "http://montage.ipac.caltech.edu/"
+    url      = "http://montage.ipac.caltech.edu/download/Montage_v5.0.tar.gz"
+
+    version('5.0', sha256='72e034adb77c8a05ac40daf9d1923c66e94faa0b08d3d441256d9058fbc2aa34')
+
+    variant('cfitsio',  default=False, description='use spack cfitsio library')
+    variant('mpi',      default=False, description='Include MPI support') 
+    variant('wcs',      default=False, description='use spack wcs library')
+
+    depends_on('cfitsio', when='+cfitsio')
+    depends_on('wcslib', when='+wcs')
+    depends_on('mpi', when='+mpi')
+
+    def patch(self):
+        if self.spec.variants['mpi'].value == 'mpi':
+            filter_file(r'#.MPICC..=', 'MPICC =', 'Montage/Makefile.LINUX')
```

Comment:
Instead of all the if conditions, could this whole patch section be changed to be more like this?

```suggestion
            filter_file(r'#.MPICC..=', 'MPICC =', 'Montage/Makefile.LINUX', when='+mpi')
```

Reply:
Also, the MPI version is not building correctly for me. I think you also need to uncomment the next line following this one in Makefile.LINUX 
```# BINS =                $(SBINS) $(MBINS)```